### ISSUE TITLE
The new product version is launched in Marketplace, when will the repo be updated?

### ISSUE THREAD

**Issue by markedcahinde** [fortinet/fortigate-autoscale-aws#8]
Hi, I'm currently doing a POC with Fortigate using this autoscaling solution. I noticed that the AMI mappings use version 6.2.3, whereas 6.4.2 is already out in the AWS Marketplace. Is this gonna be updated anytime soon? Or what's the plan in updating? Thanks!

**Comment by JaydenLiang** [comment]
@markedcahinde the support is likely to be starting from 6.4.3.

There's a way for you to try this project with 6.4.2. If you know the AMI Id for FortiGate BYOL and On-Demand 6.4.2 in a certain region, you can update the autoscale-main.template.yaml by replacing the AMI mapping. It should work.

**Comment by markedcahinde** [comment]
Thanks for the response @JaydenLiang, when will the support come out for 6.4.3?

In any case, will 6.4.2 support autoscaling as well? As with the mapping of templates, I figured I could do the update in the template, but I can't seem to hold the figures for the following


AMI:
            FGTVM64BYOL623: >-
                aws-marketplace/FortiGate-VM64-AWS build8404 (6.2.3)
                GA-e5936f4a-0d69-479f-919c-d5e158bd4d12-ami-01c5698f7658a8322.4
            FGTVM64PAYG623: >-
                aws-marketplace/FortiGate-VM64-AWSONDEMAND build8404 (6.2.3)
                GA-3124a694-441c-4ff1-8bf7-4d153be424a6-ami-0462507a813be3282.4

**Comment by JaydenLiang** [comment]
> Thanks for the response @JaydenLiang, when will the support come out for 6.4.3?
> 
> In any case, will 6.4.2 support autoscaling as well? As with the mapping of templates, I figured I could do the update in the template, but I can't seem to hold the figures for the following, esp. the GA part
> 
> AMI:
> FGTVM64BYOL623: >-
> aws-marketplace/FortiGate-VM64-AWS build8404 (6.2.3)
> GA-e5936f4a-0d69-479f-919c-d5e158bd4d12-ami-01c5698f7658a8322.4
> FGTVM64PAYG623: >-
> aws-marketplace/FortiGate-VM64-AWSONDEMAND build8404 (6.2.3)
> GA-3124a694-441c-4ff1-8bf7-4d153be424a6-ami-0462507a813be3282.4

Here they are:
aws-marketplace/FortiGate-VM64-AWS build1723 (6.4.2) GA-e5936f4a-0d69-479f-919c-d5e158bd4d12-ami-078e5e2f4b85c6733.4
and
aws-marketplace/FortiGate-VM64-AWSONDEMAND build1723 (6.4.2) GA-3124a694-441c-4ff1-8bf7-4d153be424a6-ami-072eddce6321e3dfc.4

You should be able to find this information on the **source** property of the AMI when you filter one in your EC2 console > AMI

**Comment by markedcahinde** [comment]
Thank you.
I'll check updates on the repo every now and then to verify that the support for 6.4.3 is out. 
While we're on the subject of new versions, I have tried patching and also updating using AMI from the current release of autoscaling cf templates to the latest FortiOS firmware. However, it seems that it's always getting replaced after reboot. I figured that it may be the handler that's replacing the hosts.

How can I verify that this is the case, and which CF parameter should I be updating to handle this?
